components/event-publisher-proxy/config/300-deployment.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: cloud-event-gateway-proxy
+  name: cloud-event-gateway-proxy
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: cloud-event-gateway-proxy
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: cloud-event-gateway-proxy
+    spec:
+      containers:
+        - env:
+            - name: CLIENT_ID
+              valueFrom:
+                secretKeyRef:
+                  name: cloud-event-gateway-proxy
+                  key: client-id
+            - name: CLIENT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: cloud-event-gateway-proxy
+                  key: client-secret
+            - name: TOKEN_ENDPOINT
+              valueFrom:
+                secretKeyRef:
+                  name: cloud-event-gateway-proxy
+                  key: token-endpoint
+            - name: EMS_PUBLISH_URL
+              valueFrom:
+                secretKeyRef:
+                  name: cloud-event-gateway-proxy
+                  key: ems-publish-url
+          image: github.com/kyma-project/kyma/components/cloud-event-gateway-proxy/cmd/cloud-event-gateway-proxy
```

Comment:
This needs to change to the new name. In addition, the **cloud-event-gateway-proxy** occurrences need to change too.

The same applies for the **200-service.yaml** file.

Reply:
True, changed all the occurrences.